SU-SWS/react_paragraphs
# Check for value if a user didnt select a media item

# READY FOR REVIEW

# Summary
- Prevent widget breaking if a user doesn't choose a media item.

# Need Review By (Date)
- asap

# Urgency
- high

# Steps to Test
Test these steps on `8.x-1.x` vs this branch
1. create a basic page
1. add a card item
1. click to add a media item
1. without selecting an item or uploading anything, click "insert selected"
1. Verify the widget doesn't break on this branch.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
